backend/src/permissions.js

```diff
@@ -0,0 +1,23 @@
+import { rule, shield, allow } from "graphql-shield";
+
+const isAuthenticated = rule({ cache: "contextual" })(
+  async (_parent, _args, { dataSources, id }) => {
+    let exists = dataSources.db.userExists(id);
+    return exists;
+  }
```

Comment:
toll! man kann auch mit context direkt im db.js für currentUser einstellen.

Reply:
Genau, ich war auch sehr froh, als ich rausgefunden habe, dass ich hier darauf Zugriff habe :D Wie Kirill meinte, ich finde es auch schöner so als `currentUser()` in der DB zu haben. Problem dabei wäre z.B. wenn man 2 Nutzer hat